internal/plugin/resource_bundle_install.go

```diff
@@ -468,6 +468,8 @@ only one can be configured at a time.
 					Description: docCaretToBacktick(`
 - ^artifactory.username^ (String) The Artifactory API username. This will likely be your hashicorp email address
 - ^artifactory.token^ (String) The Artifactory API token. You can sign into Artifactory and generate one
+- ^artifactory.bearer_token^ (String) The Artifactory Identity token - use instead of username/ API token. You can get a token by joining the
```

Comment:
Yeah, we should yank this. We ended up reusing token to be either the API key or the bearer token and didn't add a new attribute.